web/web.go

```diff
@@ -1100,61 +872,6 @@ func tmplFuncs(consolesPath string, opts *Options) template_text.FuncMap {
 	}
 }
 
-func (h *Handler) getTemplate(name string) (string, error) {
-	var tmpl string
-
-	appendf := func(name string) error {
-		f, err := ui.Assets.Open(path.Join("/templates", name))
-		if err != nil {
-			return err
-		}
-		defer f.Close()
-		b, err := ioutil.ReadAll(f)
-		if err != nil {
-			return err
-		}
-		tmpl += string(b)
-		return nil
-	}
-
-	err := appendf("_base.html")
-	if err != nil {
-		return "", errors.Wrap(err, "error reading base template")
-	}
-	err = appendf(name)
-	if err != nil {
-		return "", errors.Wrapf(err, "error reading page template %s", name)
-	}
-
-	return tmpl, nil
-}
-
-func (h *Handler) executeTemplate(w http.ResponseWriter, name string, data interface{}) {
-	text, err := h.getTemplate(name)
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
-	}
-
-	tmpl := template.NewTemplateExpander(
-		h.context,
-		text,
-		name,
-		data,
-		h.now(),
-		template.QueryFunc(rules.EngineQueryFunc(h.queryEngine, h.storage)),
-		h.options.ExternalURL,
-		nil,
-	)
-	tmpl.Funcs(tmplFuncs(h.consolesPath(), h.options))
```

Comment:
This was the only place where `templFuncs` was referenced, so it can be deleted as well now.

Reply:
mmm this method is used in `web_test.go`: https://github.com/prometheus/prometheus/blob/main/web/web_test.go#L93

Reply:
Yeah, that is just testing functionality that isn't used anymore, so the test can also be deleted... the whole global URL stuff is replicated (along with tests) in the API package as well, so the web copy can just go.

Reply:
alright, I removed it :)